Directory.Build.props

```diff
@@ -7,6 +7,7 @@
   <PropertyGroup>
     <Configuration Condition="'$(Configuration)'==''">Debug</Configuration>
     <Configurations>Debug;Release;Debug-Intrinsics;Release-Intrinsics</Configurations>
+    <DefineConstants>$(DefineContants);DEBUG</DefineConstants>
```

Comment:
Don't you need to set it up only if configuration contains word Debug?
Would be weird to have this constant for Release builds

Reply:
Agreed.

Reply:
If you move this line to line 116 in this file, it will just work.